src/components/Navigation/Footer.js

```diff
@@ -0,0 +1,75 @@
+import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import Grid from 'react-bootstrap/lib/Grid';
+import Row from 'react-bootstrap/lib/Row';
+
+import styles from './Footer.scss';
+
+const Footer = React.createClass({
+
+  getGitHubLink() {
+    const url = 'https://github.com/kodeklubben/oppgaver/';
+
+    return (
+      <div>
+        <Row className={styles.center}>
+          <p>Bidra?</p>
+        </Row>
+        <Row className={styles.center}>
+          <a href={url}>
+            <img className={styles.svg} src={'src/assets/graphics/github-teacher.svg'}/>
+          </a>
+        </Row>
+        <Row>
+          <div className={styles.divider}/>
+        </Row>
+      </div>
+    );
+  },
+
+  render() {
+    const url = [
+      'http://kidsakoder.no/',
+      'https://github.com/kodeklubben/kodeklubben.github.io/archive/master.zip',
+      'http://kidsakoder.no/2015/07/03/kodeklubben-trondheim-utvikler-materiell-i-sommer/'
+    ];
```

Comment:
Bare litt pirk, men hadde ikke koden blitt litt mer lesbar om man brukte et object (med key-value) i stedet for array?
F.eks.
```
const url = {
  kidsakoder: 'http://kidsakoder.no/',
  archive: 'https://github.com/kodeklubben/kodeklubben.github.io/archive/master.zip',
  sponsing: 'http://kidsakoder.no/2015/07/03/kodeklubben-trondheim-utvikler-materiell-i-sommer/'
}
```
og deretter bruke det med
```
url.kidsakoder
```